src/box/lua/schema.lua

```diff
@@ -114,12 +114,37 @@ ffi.cdef[[
     void
     port_destroy(struct port *port);
 
+    struct position {
+        uint32_t key_size;
+        const char *key;
+    };
+
+    void
+    position_reset(struct position *pos);
+
+    int
+    position_unpack(const char *ptr, struct position *pos);
+
+    const char *
+    position_pack_on_gc(struct position *pos, uint32_t *size);
+
+    int
+    box_index_tuple_position(uint32_t space_id, uint32_t index_id,
+                             const char *tuple, const char *tuple_end,
+                             struct position *pos);
+
     int
     box_select(uint32_t space_id, uint32_t index_id,
                int iterator, uint32_t offset, uint32_t limit,
                const char *key, const char *key_end,
                struct port *port);
 
+    int
+    box_select_after(uint32_t space_id, uint32_t index_id,
+                     int iterator, uint32_t offset, uint32_t limit,
+                     const char *key, const char *key_end,
+                     struct position *pos, bool update_pos, struct port *port);
+
```

Comment:
Instead of adding `box_select_after`, let's add new arguments to `box_select`. Just in case, please ask @Totktonada if we can change it.

Reply:
Done. We will ask him after his vacation.